test/e2e/PISP_Linking.test.ts

```diff
@@ -176,7 +219,12 @@ describe('Account Linking', (): void => {
     })
   })
 
-  describe('Happy Path - OTP', (): void => {
+  // the auth-service stores the consent in a database
+  // we currently do not have any endpoints to delete that consent entry atm
+  // so we can only run one set of happy path for now
+  // until we have either 1) route to delete consents or 2) more valid
```

Comment:
hmm, we need to find a way to make this easier to generate. Perhaps without requiring firebase to be set up on the pisp demo ui side.

Reply:
No real easy way around it other than generating a set of valid credentials for us to use once and done.

Which is a pain...
1) Shallow investigation of android fido apps lead me to believe the credentials don't work with the `fido2-lib` we're using OOB. They produce a `{ origin: android:apk-key-hash:some-hash-here, ... }` where `fido2-lib` is expecting a url.

Might just be my lack of knowledge on this one. So no easy emulator which can use pins/patterns.
This might be an issue if PISP's use the play store library to sign challenges.

BIG SPECULATION - I'm not knowledgable enough to predict if this will be a problem

2) On Linux you need to have a security key afaik to sign (which I and maybe other developers don't have)

Reply:
On linux, I did find this project: https://github.com/danstiner/rust-u2f, but never got it working, so I just gave up and bought a yubikey.

I'm not sure about that Android issue, but perhaps it's worth opening a ticket on our end to add to the backlog. There's probably no reason someone can't come along and get it working in the future.

For now, I think perhaps we can generate a bunch of valid and invalid credentials ahead of time, and just keep a track of them in our tests.